leo/package/src/package.rs

```diff
@@ -260,3 +367,20 @@ program {name}.aleo {{
 "#
     )
 }
+
+fn test_template(name: &str) -> String {
+    format!(
+        r#"// The 'test_{name}' test program.
+import {name}.aleo;
+program test_{name}.aleo {{
+    @test
+    script test_it() {{
+        let result: u32 = {name}.aleo/main(1u32, 2u32);
+        assert_eq(result, 3u32);
+    }}
+
+    transition do_nothing() {{ }}
```

Comment:
Should we also add a @test annotation over a transition? Maybe with @should_fail to let users know that's an option?